go/exocom-mock/exocom-mock.go

```diff
@@ -16,6 +15,7 @@ import (
 type ExoComMock struct {
 	ReceivedMessages []structs.Message
 	server           http.Server
+	serviceSockets   map[string]*websocket.Conn
```

Comment:
It was done that way over on the javascript version. Based on the features, yes it appears the map is unneeded